src/material/autocomplete/autocomplete-trigger.ts

```diff
@@ -730,12 +730,9 @@ export class MatAutocompleteTrigger
     if (toSelect) {
       this._clearPreviousSelectedOption(toSelect);
       this._assignOptionValue(toSelect.value);
-      // TODO(crisbeto): this should wait until the animation is done, otherwise the value
```

Comment:
I left this TODO, because I tried to make this change a while ago but it broke a lot of tests internally. Nothing has really changed so I doubt that we'll be able to land this PR any time soon.

Reply:
@crisbeto Would it be worth the effort to try and alter the tests? or just close this Issue?

Reply:
The issue isn't really our own tests, but tests of teams that depend on `mat-autocomplete`. We likely won't have time for a while to go through and update all of them.